src/pages/signup/signup.jsx

```diff
@@ -15,10 +17,12 @@ export function Signup() {
   const [name, setName] = useState("");
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
```

Comment:
when the user types the password in form fiedls,
make sure you hide the text input.

Reply:
Thanks for spotting that out